### DISCUSSION TITLE
Scanning Stuck on M1

### DISCUSSION THREAD

**Discussion by bhall7** [egzumer/uv-k5-firmware-custom#622] [General]
For some reason, when I start scanning, my egzumer v0.22 Quansheng uv-k5 radio just stays on memory M1. 

Here's what the display looks like after I first turn it on and it is not scanning:
![Quansheng UV-K5 NotScanning](https://github.com/user-attachments/assets/9c318ab2-fd36-42e1-8493-3288240b20b9)

Here's what the display looks like after I activate scan mode. It stays stuck on channel M1:
![Quansheng UV-K5 Scanning](https://github.com/user-attachments/assets/a1312087-fb5c-406e-a599-be6e00c113b2)

Scanning was working previously, but no matter what I do, I cannot get the radio to scan through all of my memory channels like before. What am I missing? Thanks!

**Comment by prokrypt** [comment]
Add each channel into the appropriate scan list.

**Comment by bhall7** [comment]
I figured it out. Somehow, in the menu, under SList, it was set to LIST1, but after changing it to ALL, it started working as before. 🙂
